website/sidebars.js

```diff
@@ -164,6 +164,7 @@ module.exports = {
             'fb/internal-fb-cluster',
             'fb/fair-cluster',
             'fb/fbcode-configerator-config-source',
+            'fb/flow-launcher',
```

Comment:
You probably also need to manually add it to the versioned sidebar snapshot for 1.0.

Reply:
did you mean [here](https://github.com/facebookresearch/hydra/blob/master/website/versioned_sidebars/version-1.0-sidebars.json#L369-L374)? 

I don't see anything here, am I looking at the wrong place?

Reply:
same file, in fbcode. (it's excluded from the sync).

Reply:
right, thanks! I vaguely remember I had to do something extra last time I did this, will note this down :)